packages/azure-themes/src/azure/styles/DetailsList.styles.ts

```diff
@@ -153,19 +153,26 @@ export const DetailsRowStyles = (props: IDetailsRowStyleProps): Partial<IDetails
         {
           backgroundColor: extendedSemanticColors.listItemBackgroundSelected,
           textDecoration: extendedSemanticColors.listUnderline,
+          fontWeight: '400',
```

Comment:
It might be better to create enum variable for font weights and reference it like FontWeights.weight4. Color and font sizes use variables.